src/ui/components/RecommendedBadge/index.js

```diff
@@ -31,7 +31,7 @@ export const RecommendedBadgeBase = ({ i18n, size }: InternalProps) => {
     >
       <a
         className="RecommendedBadge-link"
-        href="https://support.mozilla.org/"
+        href="https://support.mozilla.org/en-US/kb/recommended-extensions-program"
```

Comment:
Let's use a URL without the locale so that it redirects for the user: `https://support.mozilla.org/kb/recommended-extensions-program`

Reply:
Oops. Thanks for catching that!